src/browser-modules/eval.js

```diff
@@ -12,15 +12,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 module.exports = function(code, fileName) {
     var res = {};
     // Using the 'arguments[1].res = ...' trick because IE does not let eval return a function
     if (fileName) {
-        code = ['/*\n * File: ', fileName, '\n */\narguments[1].res=', code, '\n//# sourceURL=', fileName].join('');
+        // Compute sourceURL
+        var sourceURL = "";
+        if (window.location && typeof window.location.origin === "string") {
+            sourceURL = window.location.origin + "/";
+        }
+        // Normalize fileName format 
+        if (fileName.indexOf('/') === 0) {
+            sourceURL += fileName.substring(1);
+        } else {
+            sourceURL += fileName;
+        }
+
+        code = ['/*\n * File: ', fileName, '\n */\narguments[1].res=', code, '\n//# sourceURL=', sourceURL].join('');
```

Comment:
I would not do these changes in this file, because it is possible that `fileName` is already a full URL. It would be better to change things before in the call stack.

Reply:
To match you comment I think the best is to test the provided URL againt a location RegExp, what do you think ?

Reply:
hmm then I'm thinking again about your comment, that would be better to do it directly in AT

Reply:
@divdavem what do you think if I make the change directly in https://github.com/ariatemplates/ariatemplates/blob/master/src/aria/Aria.js#L1651 ?
With a RegExp I can check if srcURL don't match an absolute URL pattern, in that case I add the current location to the URL.